odoo/orm/models.py

```diff
@@ -3483,18 +3556,21 @@ def _check_access(self, operation: str) -> tuple[Self, Callable] | None:
         and :meth:`_filtered_access`. The method may be overridden in order to
         restrict the access to ``self``.
         """
-        Access = self.env['ir.model.access']
-        if not Access.check(self._name, operation, raise_exception=False):
-            return self, functools.partial(Access._make_access_error, self._name, operation)
+        env = self.env
+        # get the cached access
+        ModelAccess = env['ir.model.access']
+        if not ModelAccess.check(self._name, operation, raise_exception=False):
+            return self, functools.partial(ModelAccess._make_access_error, self._name, operation)
```

Comment:
Does it actually make a difference?  If it ain't broken, don't fix it...

Reply:
It was just to align naming as part of the function changed.

Reply:
I will rollback these changes.